packages/compass-data-modeling/src/components/diagram-editor-side-panel.tsx

```diff
@@ -54,15 +37,21 @@ function DiagmramEditorSidePanel({
   }
 
   return (
-    <div
-      className={cx(containerStyles, isDarkMode && darkModeContainerStyles)}
-      data-testid="data-modeling-drawer"
+    <DrawerSection
+      id={DATA_MODELING_DRAWER_ID}
+      title="Details"
+      label="Details"
+      glyph="InfoWithCircle"
+      autoOpen
+      // TODO: Leafygreen doesn't allow us to tie close event to a particular
+      // action. We can add this functionality ourselves, but I'm not sure that
+      // adding even more logic on top of the drawer is a good idea. Maybe we're
+      // okay with the drawer close button click just staying there until you
+      // explicitly click something else?
+      // onClose={onClose}
```

Comment:
That's kinda the only thing missing here, but as mentioned, I think it might be okay to just skip to avoid adding extra complexity to this system. Literally the next project for Data Modeling will make this redundant anyway because we will always have at least one section in the drawer (no selection becomes sort of "top level" selection) and so this button will behave as just "collapse" not "deselect" anyway.

I also can't see a use-case for providing this onClose interface in "AI Assistant" project